From 17a03a09c180867a48256bf31b747b004008b1db Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Tue, 5 May 2026 13:30:14 +0800
Subject: [PATCH] dts: arm: move ast2705-evb.dtb under CONFIG_ASPEED_AST2705

ast2705-evb.dtb should be guarded by the new CONFIG_ASPEED_AST2705
symbol rather than CONFIG_ASPEED_AST2700.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: I9e3c81f61c5b348c1aa10f4a8839379bd84adcd1
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 arch/arm/dts/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 29ce14ffd9..4195608f54 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -1285,13 +1285,15 @@ dtb-$(CONFIG_ASPEED_AST2600) += \
 
 dtb-$(CONFIG_ASPEED_AST2700) += \
 	ast2700-evb.dtb \
-	ast2705-evb.dtb \
 	ast2700-raw.dtb \
 	ast2700-slt.dtb \
 	ast2700-ncsi.dtb \
 	ast2700-dcscm.dtb \
 	ast2700-vff.dtb
 
+dtb-$(CONFIG_ASPEED_AST2705) += \
+	ast2705-evb.dtb
+
 dtb-$(CONFIG_ARCH_STI) += stih410-b2260.dtb
 
 dtb-$(CONFIG_STM32MP13x) += \
-- 
2.43.0

